src/core/plugins/json-schema-2020-12/fn.js

```diff
@@ -164,7 +165,7 @@ export const makeGetType = (fnAccessor) => {
     return combinedStrings || "any"
   }
 
-  return getType
+  return (schema, ...rest) => getType(immutableToJS(schema), ...rest)
```

Comment:
Let's handle this in `oas31/after-load`
```suggestion
  return getType
```